template.yaml

```diff
@@ -90,6 +90,7 @@ Resources:
       Role: !GetAtt TriggerExecutionRole.Arn
       MemorySize: 1408
       Timeout: 10
+      ReservedConcurrentExecutions: 3
```

Comment:
@janbj @svenngjeto is there a possibility to test this somehow? If not let's merge it?

Reply:
According to @brinxmat  and @svenngjeto  we need to increase the memory to 1792 MB for all services related to user handling so that the lambdas will get a dedicated CPU when they run.  Increasing the memory will slow up start-up time.  Package size is also important.

Reply:
Isn't it the `ProvisionedConcurrencyConfig` that we should be looking into? At least that is what i have found out:
https://aws.amazon.com/blogs/compute/new-for-aws-lambda-predictable-start-up-times-with-provisioned-concurrency/

I think ReservedConcurrency affects how many instances we can have in total.

Reply:
@axthosarouris : We must figure out what is correct way to configure this. From documentation for cloudformation templates I have this link:

https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html#sam-function-provisionedconcurrencyconfig  (link shows information about `ProvisionedConcurrencyConfig` and `ReservedConcurrentExecutions`

This describes how to configure the `ProvisionedConcurrencyConfig` :
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-alias-provisionedconcurrencyconfiguration.html

and `ReservedConcurrentExecutions` is: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-reservedconcurrentexecutions

From what I can understand:

`ReservedConcurrentExecutions`: The number of simultaneous executions to reserve for the function.

`ProvisionedConcurrentExecutions` (under `ProvisionedConcurrencyConfiguration` ): The amount of provisioned concurrency to allocate for the alias.

I believe we actually want to configure `ProvisionedConcurrentExecutions` , as of now we don't have any need for configure a maximum concurrency as well. 

I will push an update @axthosarouris